optaplanner-persistence/pom.xml

```diff
@@ -32,4 +32,16 @@
     <module>optaplanner-persistence-jsonb</module>
   </modules>
 
+  <!-- TODO: Move to optaplanner-build-parent once https://github.com/quarkusio/quarkus-platform-bom-generator/issues/64 is solved.  -->
+  <dependencyManagement>
+    <dependencies>
+      <!-- TODO Dependencies that should maybe be removed/replaced in 8 https://issues.redhat.com/browse/PLANNER-2262 -->
+      <dependency>
+        <groupId>org.eclipse.persistence</groupId>
+        <artifactId>org.eclipse.persistence.moxy</artifactId>
```

Comment:
We use moxy to test `JaxbAdapter`s for `Score`; specifically, to test their (un)marshalling to JSON via JAXB.

If we want to drop this dependency, there are two options:
A) give up the JAXB - JSON test coverage
B) use the jackson-module-jaxb-annotations instead, so that (un)marshalling is done via Jackson, but still using JAXB annotations

Reply:
I honestly never understood why we needed Moxy in the first place. In my own projects, I have been able to deserialize JSON into JAXB w/ RESTEasy, w/ Jackson integration.

Reply:
I suppose that incorporates the option B) under the hood. According to [1], the JAXB reference implementation does not support JSON.

[1] https://stackoverflow.com/questions/38789307/how-to-serialize-jaxb-object-to-json-with-jaxb-reference-implementation